js/app/time.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * time.js
+ *
+ * made by Rick Nienhuis & Niels Haan
+ */
+
+define(function() {
+
+	var date, year, day, month, hours, minutes, seconds;
+
+	function getTizenDateTime() {
+		var date;
+		try {
+			date = tizen.time.getCurrentDateTime();
+		} catch (err) {
+			console.error('Error: ', err.message);
+		}
+		return date;
+	}
+
+	return {
+
+		create: function() {
+			date = getTizenDateTime();
+			year = date.getFullYear();
+			day = date.getDate();
+			month = date.getMonth() + 1;
+			hours = date.getHours();
+			minutes = date.getMinutes();
+			seconds = date.getSeconds();
+
+			if (month < 10) {
```

Comment:
Do not repeat yourself!
create one function:
`function xyz(value) {
   if (value < 10)
       return "0" + value;
return value
}
`